third_party/react-day-picker/style.css

```diff
@@ -0,0 +1,310 @@
+:root {
```

Comment:
This is copy/pasted from react-day-picker. I'm not sure whether we can import this directly from the library, or whether we still need to maintain our own copy here.